corehq/apps/reports/templates/reports/partials/data_corrections_trigger.html

```diff
@@ -1,6 +0,0 @@
-{% load i18n %}
-
-<button type="button" class="btn btn-default pull-left data-corrections-trigger">
-    <i class="fa fa-edit"></i>
-    {% trans 'Data Corrections' %}
-</button>
```

Comment:
not related to this PR but I see you're removing it here anyway: https://github.com/dimagi/commcare-hq/pull/20587
Not going to merge though since I think merging this will cause a conflict on the other one which is dependent on this PR.

Reply:
yeah, got my wires crossed, meant to do this in the other PR